tx-source/site_core.php

```diff
@@ -1667,6 +1668,7 @@
 $Definition['via %s'] = 'via %s';
 $Definition['Video embedding has been disabled.'] = 'Video embedding has been disabled. URLs will not translate to their embedded equivalent.';
 $Definition['View'] = 'View';
+$Definition['View:'] = 'View:';
```

Comment:
Could we add a the colen separately? It feels like a UI/decorative element, rather than part of the text.